server/public/shared/i18n/i18n.go

```diff
@@ -113,7 +113,7 @@ func GetTranslationFuncForDir(dir string) (TranslationFuncByLocal, error) {
 	}, nil
 }
 
-func getTranslationsBySystemLocale() (TranslateFunc, error) {
+func GetTranslationsBySystemLocale() (TranslateFunc, error) {
```

Comment:
In order to be able to get the default locale I had to make this accessible. This is so that Wrangler can render its messages in the system default locale, rather than relying on the initiating user's locale (messages are then translated client side to each user's locale)